specification/paloaltonetworks/resource-manager/PaloAltoNetworks.Cloudngfw/preview/2024-01-19-preview/PaloAltoNetworks.Cloudngfw.json

```diff
@@ -0,0 +1,7564 @@
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Azure operation completed successfully.",
+            "schema": {
+              "$ref": "#/definitions/FirewallResourceListResult"
+            }
+          },
+          "default": {
+            "description": "Common error response for all Azure Resource Manager APIs to return error details for failed operations.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "Firewalls_ListBySubscription_MaximumSet_Gen": {
+            "$ref": "./examples/Firewalls_ListBySubscription_MaximumSet_Gen.json"
+          },
+          "Firewalls_ListBySubscription_MinimumSet_Gen": {
+            "$ref": "./examples/Firewalls_ListBySubscription_MinimumSet_Gen.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/PaloAltoNetworks.Cloudngfw/listCloudManagerTenants": {
+      "post": {
+        "operationId": "PaloAltoNetworksCloudngfw_listCloudManagerTenants",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request has succeeded.",
+            "schema": {
+              "type": "array",
+              "items": {
+                "type": "string"
+              },
+              "x-cadl-name": "string[]"
+            }
+          },
+          "default": {
+            "description": "Common error response for all Azure Resource Manager APIs to return error details for failed operations.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "PaloAltoNetworksCloudngfw_listCloudManagerTenants_MaximumSet_Gen": {
+            "$ref": "./examples/PaloAltoNetworksCloudngfw_listCloudManagerTenants_MaximumSet_Gen.json"
+          },
+          "PaloAltoNetworksCloudngfw_listCloudManagerTenants_MinimumSet_Gen": {
+            "$ref": "./examples/PaloAltoNetworksCloudngfw_listCloudManagerTenants_MinimumSet_Gen.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/PaloAltoNetworks.Cloudngfw/listSupportAccountStatus": {
+      "post": {
```

Comment:
nit: pagination not required for these list APIs ?

Reply:
Made it a model